platform/containers/axclustermanager/Dockerfile

```diff
@@ -29,4 +29,12 @@ COPY motd /etc/motd
 RUN echo '[ ! -z "$TERM" -a -r /etc/motd ] && cat /etc/motd' >> /etc/bash.bashrc
 RUN echo 'PS1="argo cluster ops> "' >> /root/.bashrc
 
+# some pip installs
+# TODO: Need to move pip installs here so that it does not have dependency with kubernetes-distro container
+RUN pip install anytree==2.2.1 prompt-toolkit==1.0.15
+
+WORKDIR /ax/bin
+
+ENTRYPOINT ["bash", "-c", "cat /etc/motd && python -m clustercli.main_cli"]
```

Comment:
can the container still be programmable? i.e. portal needs do things such as
```
docker run <container> argocluster install xxx
```
Though we are modifying entry point for user container in workflow, but since this entrypoint is interactive and blocking, how would this work?

Reply:
good point. This is going to affect those commands. I could change this to using ENTRYPOINT to CMD and then you can use docker run commands...